libraries/helpers.rb

```diff
@@ -71,7 +71,7 @@ def extension_installed?(new_resource)
       if new_resource.version
         version_result.stdout == new_resource.version
       else
-        !version_result.stdout.nil?
+        !version_result.stdout.eql?("\n")
```

Comment:
We should really be stripping `/n` rather than making sure something equals an invalid character

Reply:
I'd look at using `.chomp` to strip out extra whitespace which will give you a true on `empty?`

```ruby
"\n".chomp
=> ""
irb(main):002:0> "\n".chomp.empty?
=> true
```